redhat-developer/app-services-cli
# WIP: feat(status) display failed_reason for a failing Kafka instance

### Description

Display `failed_reason` in status of Kafka instances that are failing.

![Screenshot from 2021-03-22 12-18-04](https://user-images.githubusercontent.com/23582438/111953959-61686780-8b0d-11eb-8414-f8018c161f25.png)

Fixes #474 

### Verification Steps

1. Select a Kafka instance that has failed status.
> go run ./cmd/rhoas kafka use <instance-name>
2. Run `status` command to see the status.
> go run ./cmd/rhoas status

### Type of change


### Checklist
